docs/2.8.0/docs/kubernetes/02-provision-cloud-resources/01-provision-cloud-resources.rst

```diff
@@ -126,49 +126,49 @@ There are different ways to configure access to the Kubernetes API using the CLI
 a) Using a kubectl configuration context (recommended)
 ------------------------------------------------------
 
-* Open new terminal to make an SSH tunnel (available on ``localhost:44443``\ ):
+* Open a new terminal window to make an SSH tunnel (available on ``localhost:44443``\ ):
 
   .. code-block:: bash
 
      make kubectl-proxy
 
-* Merge and switch to the new ``kubectl`` configuration (aka ``kubeconfig``\ ) to your AKS cluster using the SSH tunnel:
+* Merge and switch to the new ``kubectl`` configuration (also called ``kubeconfig``\ ) to your AKS cluster using the SSH tunnel:
 
   .. code-block:: bash
 
      ./bootstraps/merge-kubeconfig.sh
 
-* Run commands ``kubectl``\ , ``helm``\ , ``helmfile``\ , etc. as usual
+* Run commands such as ``kubectl``\ , ``helm``\ , and ``helmfile``\  as usual.
 
-b) Targeting the kubectl configuration file
--------------------------------------------
+b) Target the kubectl configuration file
+----------------------------------------
 
-* Open new terminal to make an SSH tunnel (available on ``localhost:44443``\ ):
+* Open a new terminal window to make an SSH tunnel (available on ``localhost:44443``\ ):
 
   .. code-block:: bash
 
      make kubectl-proxy
 
-* Set Kubernetes CLI configuration file (lost if you use a new/different terminal)
+* Set the  Kubernetes CLI configuration file (this is lost if you use a new or different terminal):
 
   .. code-block:: bash
 
      export KUBECONFIG="$(pwd)/outputs/kube/ssh.config"
 
-* Run commands ``kubectl``\ , ``helm``\ , ``helmfile``\ , etc. as usual
+* Run commands such as ``kubectl``\ , ``helm``\ , and ``helmfile``\ as usual.
 
 Access to PostgreSQL
 ====================
 
-To enable accessing to the provisioned PostgreSQL server:
+To enable access to the provisioned PostgreSQL server:
 
-* Open a new terminal to make an SSH tunnel (available on ``localhost:5432``\ )
+* Open a new terminal window to make an SSH tunnel (available on ``localhost:5432``\ )
```

Comment:
I would leave it as "terminal" without the "window" but not a big deal either way